keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -0,0 +1,270 @@
+  - [Notes/Constraints/Caveats (optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Phase 0](#phase-0)
+    - [needs-triage and triage/accepted-by-sig labels](#needs-triage-and-triageaccepted-by-sig-labels)
+    - [Remove or rename unused triage/** labels](#remove-or-rename-unused-triage-labels)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+- [Infrastructure Needed](#infrastructure-needed)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [ ] Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] KEP approvers have approved the KEP status as `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] Graduation criteria is in place
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://github.com/kubernetes/enhancements
+[kubernetes/kubernetes]: https://github.com/kubernetes/kubernetes
+[kubernetes/website]: https://github.com/kubernetes/website
+
+## Summary
+
+To ease the burden of SIG/area reviewers/approvers, we would like to prescribe
+a triage workflow and supporting automation.
+
+### Note to Reviewers
+
+For this `provisional` phase, the details of this KEP are left intentionally
+light. There was significant discussion across issues, mailing list threads,
+and PRs that didn't lead to forward progress because it seemed we were trying
+to solve everything at once.
+
+Here we attempt to scope a single deliverable before moving on to discussing
+workflow, label states, and entry/exit criteria.
+
+## Motivation
+
+At present, there are [2,155 open issues and 902 pull requests](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen) open in
+`kubernetes/kubernetes`.
+
+- [356 are categorized as `lifecycle/stale`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Fstale)
+- [246 are categorized as `lifecycle/rotten`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Frotten)
+- [690 are categorized as `lifecycle/frozen`](https://github.com/kubernetes/kubernetes/labels/lifecycle%2Ffrozen)
+
+This makes for about 20% of open issues/PRs that are in some state of staleness.
+If we consider items marked as `lifecycle/frozen`, then we're looking at around
+42% of issues/PRs that could potentially require attention.
+
+### Goals
+
+See [User Stories](#user-stories).
+
+<unresolved>
```

Comment:
`[[<unresolved>]]` ?

Reply:
Heh. `<<[UNRESOLVED]>>`
ref: https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/617-improve-kep-implementation#1-standardize-an-intra-kep-denotation-for-not-resolved